packages/web-runtime/tests/unit/components/ApplicationsMenu.spec.js

```diff
@@ -0,0 +1,146 @@
+import { shallowMount, createLocalVue } from '@vue/test-utils'
+import Vuex from 'vuex'
+import DesignSystem from 'owncloud-design-system'
+import GetText from 'vue-gettext'
+import VueRouter from 'vue-router'
+
+import navigation from 'web-runtime/src/store/navigation'
+import ApplicationsMenu from 'web-runtime/src/components/ApplicationsMenu.vue'
+
+const localVue = createLocalVue()
+localVue.use(Vuex)
+localVue.use(VueRouter)
+localVue.use(DesignSystem)
+localVue.use(GetText, {
+  translations: 'does-not-matter',
+  silent: true
+})
+
+const menuInternalLink = {
+  name: 'Files',
+  id: 'files',
+  icon: 'folder',
+  type: 'extension'
+}
+
+const menuExternalLink = {
+  name: 'External',
+  id: 'external',
+  icon: 'some-icon',
+  type: '',
+  url: 'http://some.org'
+}
+
+const staticNavItems = [
+  {
+    name: 'All files',
+    iconMaterial: 'folder',
+    route: {
+      name: 'files-personal',
+      path: '/files/list/all'
+    }
+  },
+  {
+    name: 'Favorites',
+    iconMaterial: 'star',
+    route: {
+      name: 'files-favorites',
+      path: '/files/list/favorites'
+    }
+  },
+  {
+    name: 'Shared with me',
+    iconMaterial: 'shared-with-me',
+    route: {
+      name: 'files-shared-with-me',
+      path: '/files/list/shared-with-me'
+    }
+  },
+  {
+    name: 'Shared with others',
+    iconMaterial: 'shared-with-others',
+    route: {
+      name: 'files-shared-with-others',
+      path: '/files/list/shared-with-others'
+    }
+  },
+  {
+    name: 'Shared via link',
+    iconMaterial: 'link',
+    route: {
+      name: 'files-shared-via-link',
+      path: '/files/list/shared-via-link'
+    }
+  },
+  {
+    name: 'Deleted files',
+    iconMaterial: 'delete',
+    route: {
+      name: 'files-trashbin',
+      path: '/files/list/trash-bin'
+    }
+  }
+]
+
+const selectors = {
+  appSwitcherButton: '#_appSwitcherButton',
+  internalMenuLink: '[data-testid="apps-menu-internal-link"]',
```

Comment:
These are supposed to check for internal links (vue routes) and external links (a tags referencing outside the `web` app). Could you refactor the tests to use both and check they work?